src/main/java/com/thoughtworks/aceleradora/dominio/Nivel.java

```diff
@@ -0,0 +1,44 @@
+package com.thoughtworks.aceleradora.dominio;
+
+public class Nivel {
+    private int ordem;
+    private String nome;
+
+    public Nivel() {
+    }
+
+    public Nivel(int ordem, String nome) {
+        this.ordem = ordem;
+        this.nome = nome;
+    }
+
+    public int getOrdem() {
+        return ordem;
+    }
+
+    public String getNome() {
+        return nome;
+    }
+
+    public void setOrdem(int ordem) {
+        this.ordem = ordem;
+    }
+
+    public void setNome(String nome) {
+        this.nome = nome;
+    }
+
+    @Override
+    public String toString() {
+        return "Ordem: " + ordem + ", Nome: " + nome;
+    }
+
+    @Override
+    public Object clone() {
```

Comment:
A ideia de ter uma implementação de clone é legal, mas eu não sei se estamos fazendo da melhor maneira. Aparentemente, esse método sempre retorna `null`. A classe `Nivel` não implementa `Cloneable`, o que sempre irá gerar `CloneNotSupportedException`.

https://stackoverflow.com/questions/18108414/what-type-of-copy-does-the-default-java-lang-object-clone-method-perform